build error on FreeBSD & CLang

I tried update the herbstluftwm port on FreeBSD to 0.8.0 but I ran into this error:
```
/wrkdirs/usr/ports/x11-wm/herbstluftwm/work/herbstluftwm-0.8.0/src/completion.cpp:38:33: error: invalid operands to binary expression ('basic_ostream<char, std::__1::char_traits<char> >' and
'const char *')
        output_ << escape(word) << (shellOutput_ ? " \n" : "\n");
....
/wrkdirs/usr/ports/x11-wm/herbstluftwm/work/herbstluftwm-0.8.0/src/completion.cpp:75:33: error: invalid operands to binary expression ('basic_ostream<char, std::__1::char_traits<char> >' and
'const char [2]')
        output_ << escape(word) << "\n";

```

--- a/src/completion.cpp
+++ b/src/completion.cpp
@@ -1,6 +1,7 @@
 #include "completion.h"
 
 #include <algorithm>
+#include <iostream>
 #include <string>
 #include <vector>
 

